src/modules/keyboardmanager/ui/EditKeyboardWindow.cpp

```diff
@@ -214,11 +307,14 @@ void createEditKeyboardWindow(HINSTANCE hInst, KeyboardManagerState& keyboardMan
 
     // Main Header Apply button
     Button applyButton;
-    applyButton.Content(winrt::box_value(L"Apply"));
-    header.SetAlignRightWithPanel(applyButton, true);
-    header.SetLeftOf(cancelButton, applyButton);
+    applyButton.Content(winrt::box_value(L"OK"));
```

Comment:
Can we also set a width for both the OK and Cancel button to be same, and the same width can be used for shortcuts page as well.
![image](https://user-images.githubusercontent.com/32061677/81461403-53f9f700-9160-11ea-8a2a-669cd0ddb6eb.png)
@saahmedm what are your thoughts on this? It's a quick change of just adding a width property so we can make it match like we did for the Type Key and Type Shortcut in #2798 . I think the size of the Cancel button is fine, so we just find an exact size value similar to that and put the same for OK as well.

Reply:
Agree here